### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by satyampsoni**
Added contributions section

<!--  Thanks for sending a pull request!  Here are some tips for you:

This PR improves the Readme.md file.
We don't have contributing section to readme.md in order to help beginners get started with contributions.

┆Issue is synchronized with this [Jira Task](https://gr-oss.atlassian.net/browse/BATCH-290) by [Unito](https://www.unito.io)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -54,6 +54,10 @@ We expect readers of the documentation to have a basic understanding of Docker a
 - [Docker overiew](https://docs.docker.com/get-started/overview/)
 - [Kubernetes overview](https://kubernetes.io/docs/concepts/overview/)
 
+## Contributions
+Thank you for considering contributing to Armada! We want everyone to feel that they can contribute to the Armada Project.Your contributions are valuable, whether it's fixing a bug, implementing a new feature, improving documentation, or suggesting enhancements. We appreciate your time and effort in helping make this project better for everyone.
```
Comment (kannon92) [inline README.md:58]:
```suggestion
Thank you for considering contributing to Armada!  
We want everyone to feel that they can contribute to the Armada Project.  
Your contributions are valuable, whether it's fixing a bug, implementing a new feature, improving documentation, or suggesting enhancements.  
We appreciate your time and effort in helping make this project better for everyone.
```

Comment (kannon92) [reply README.md]: Ask yourself the question: What do I wish I would've seen when starting development in Armada?  Was there anything you found confusing?

Maybe post some ideas in the #2182.

**Review by kannon92 — DISMISSED** [review]

File: README.md (hunk 1 above)
Comment (satyampsoni) [reply README.md]: I have added a pull request today only adding `good-first` to contributing.md. 
Here is the link to the PR would love to add the approach for Armada development too there. 
https://github.com/armadaproject/armada/pull/2479. Pls suggest the changes and I'll do that accordingly.

**Review by kannon92 — DISMISSED** [review]

**Comment by satyampsoni** [comment]
Sorry, got confused over the branches. I have deleted those extra branches, is it fine now?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -64,6 +64,11 @@ Real-time interactions between Armada developers and users occurs primarily in C
 * If you already have an account on CNCF Slack, join #armada on [https://cloud-native.slack.com](https://cloud-native.slack.com).
 * If you need an inviation to CNCF Slack, you can get one at [https://slack.cncf.io](https://slack.cncf.io).
 
+## Finding Issues to Work On
+If you're new to the project and looking for a place to start, we recommend checking out the issues tagged with "good first". These issues are specifically curated for newcomers to the project, providing an opportunity to get familiar with the codebase and make meaningful contributions.
+
+You can view the list of ["good first"](https://github.com/armadaproject/armada/labels/good%20first%20issue) issues.
```
Comment (kannon92) [inline CONTRIBUTING.md:70]:
```suggestion
You can view the list of [good first issues](https://github.com/armadaproject/armada/labels/good%20first%20issue) issues.
```

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -64,6 +64,11 @@ Real-time interactions between Armada developers and users occurs primarily in C
 * If you already have an account on CNCF Slack, join #armada on [https://cloud-native.slack.com](https://cloud-native.slack.com).
 * If you need an inviation to CNCF Slack, you can get one at [https://slack.cncf.io](https://slack.cncf.io).
 
+## Finding Issues to Work On
+If you're new to the project and looking for a place to start, we recommend checking out the issues tagged with "good first". These issues are specifically curated for newcomers to the project, providing an opportunity to get familiar with the codebase and make meaningful contributions.
```
Comment (kannon92) [inline CONTRIBUTING.md:68]:
```suggestion
If you're new to the project and looking for a place to start, we recommend checking out the issues tagged with "good first issues". These issues are specifically curated for newcomers to the project, providing an opportunity to get familiar with the codebase and make meaningful contributions.
```

**Review by kannon92 — APPROVED** [review]
Awesome!

**Pull request merged**